staging/src/k8s.io/apiserver/pkg/endpoints/handlers/responsewriters/errors.go

```diff
@@ -48,8 +48,10 @@ func Forbidden(attributes authorizer.Attributes, w http.ResponseWriter, req *htt
 
 func forbiddenMessage(attributes authorizer.Attributes) string {
 	username := ""
+	groups := []string{}
```

Comment:
This information is printed internally by the API server (at least for RBAC): https://github.com/kubernetes/kubernetes/blob/v1.6.0-rc.1/plugin/pkg/auth/authorizer/rbac/rbac.go#L87

Maybe that's sufficient?